### PR TITLE
Make base for GO import

### LINKED ISSUE #3057: Change GO import to make base instead of use base
Having a close look into `go-base.owl` file we import and it comes with a long list of RO properties, all with their own annotations.

image

### PR DESCRIPTION
**Pull request by anitacaron**
~The artefact `go-base.owl` is in reality a base plus artefact (base + the inferred axioms). This reduces the size of the `merged_import.owl` by half.~

Fixes #3057

### CODE REVIEW

File: src/ontology/uberon-odk.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ import_group:
     - id: cl
       use_base: TRUE
     - id: go
-      use_base: TRUE
+      make_base: TRUE
```
Comment (matentzn) [inline src/ontology/uberon-odk.yaml:47]:
```suggestion
      make_base: TRUE
      mirror_from: http://purl.obolibrary.org/obo/go/go-base.owl
```

Comment (matentzn) [reply src/ontology/uberon-odk.yaml:47]: unfortunately, you have not been properly introduced to the GO ontology structure. What is sold on http://purl.obolibrary.org/obo/go.owl is a drastically reduced version of GO which is lacking a lot of the rich links to other ontologies. So you need to add _mirror_from_ as well here. http://purl.obolibrary.org/obo/go/go-base.owl has _more_, not _less_ then http://purl.obolibrary.org/obo/go.owl.. Gr.. Sorry. 

In any case, hopefully, you will still get _some_reduction, just not as much as you are hoping :)

**Review by matentzn — CHANGES_REQUESTED** [review]

**Review by matentzn — COMMENTED** [review]
Before I confirm, lets wait and see how the import module looks like - QC should run over the correct version..

File: src/ontology/uberon-odk.yaml (hunk 1 above)
Comment (anitacaron) [reply src/ontology/uberon-odk.yaml:47]: but we don't need the link to other ontologies when importing GO, right?

Comment (matentzn) [reply src/ontology/uberon-odk.yaml:47]: They drive inference! So those axioms will contribute to reasoning of terms involving GO terms (especially taxon stuff, but also many other things). So yes, conceptually at least we need them. Practically, no one knows.

Comment (anitacaron) [reply src/ontology/uberon-odk.yaml:47]: yeah, I didn't see any difference for Uberon when using the `go.owl`, but that's fine.

File: src/ontology/imports/merged_import.owl [hunk 2]
```diff
@@ -12250,35 +12253,26 @@ AnnotationAssertion(rdfs:isDefinedBy <http://purl.obolibrary.org/obo/IAO_0000232
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/IAO_0000232> "curator note"@en)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/IAO_0000232> "curator note")
 
-# Annotation Property: <http://purl.obolibrary.org/obo/IAO_0000233> (term tracker item)
```
Comment (matentzn) [inline src/ontology/imports/merged_import.owl:12253]: Seems a very risky change.

**Review by matentzn — APPROVED** [review]

**Pull request merged**